product_docs/docs/transporter/ba_preview/getting_started/config_reader.mdx

```diff
@@ -72,37 +72,41 @@ Set the following environment variables in `/opt/cdcreader/run-cdcreader.sh` wit
 #export QUARKUS_LOG_CATEGORY__COM_ENTERPRISEDB__LEVEL=DEBUG
 ```
 
-## Running the reader
```

Comment:
Moved this to the bottom of the page, as this is the command the customer will have to run _after_ configuring all values.